tom/tag.py

```diff
@@ -63,6 +63,9 @@ def get_next_build_tag(self, old_tag, branch):
                 raise TagParsingException('Could not split tag [%s] into parts' % old_tag)
         else:
             # 3.15.0 -> 3.16.0-build1 or 3.15.1-build1, depending on branch
+            # But first, strip release number from version
+            # 3.12.5-2 -> 3.12.5
+            old_tag = re.sub('-.*','',old_tag)
```

Comment:
Is this expected to only remove `-[0-9]+`? If so, make the regex more specific, please.